### ISSUE TITLE
Testing Field Maps App - February 2024 Holistic Testing - New Layer for Tasks

### ISSUE THREAD

**Issue by pjdohertygis** [pjdohertygis/SARCOP#245]
**Instructions for Installation:** 
New tech preview offerings for ArcGIS Field Maps are now available. Please follow the directions below to install builds on your mobile device(s).

**Send Jared and Paul your commonly used ArcGIS Online username if you want to be added to the dropdown list for Tasks assignments. **

**Additional Functionality**
As of September 2024, Esri has released additional functionality we would like you to test. 
- Personal mark-up (the ability to annotate the map and airdrop to friends)
- SharePlay (iOS only the ability to FaceTime and share your location on the map)
- Geofencing alerts (going in and out of segments, severe weather warnings)

**Android**
You can access the [APK here](https://esriis-my.sharepoint.com/:u:/g/personal/jeffs_esri_com/EQY7iVV7gmVPpz2WvFwjCAEBVchjvL_mYCUFkE-1-_CyrA?e=IkpcEe).  Password: TechPreview3-2024
The Android tech preview build number is 1313 and you can find the build number from the About page within your Profile. 

**iOS**
Download and install the [TestFlight app](https://itunes.apple.com/us/app/testflight/id899247664?mt=8) from the Apple App Store.
Copy [this link](https://testflight.apple.com/join/eVbcZEAZ) (https://testflight.apple.com/join/eVbcZEAZ) to your iOS device (perhaps by sending yourself an email) or by scanning the QR code below:
![image](https://github.com/user-attachments/assets/d6adfdd5-a067-4214-ae45-ddd951952bbe)

Tap on the link and it should directly open the TestFlight app on your device.
Tap Install to install the ArcGIS Field Maps app.
The iOS tech preview build number is 1214 and you can find the build number from the About page within your Profile.

Note: the TestFlight app should notify you when new builds are available. To update to the latest build, simply tap Update in the TestFlight app.

**Instructions for Testing**
Once the install is complete, test with this web map: 
1) Open Field Maps Preview (warning, this is different from the app store and Beta version)
![image](https://github.com/pjdohertygis/WideAreaSearchTemplates/assets/4256979/0b7200a4-c5a6-43e3-b816-1b13f2aa5b66)
2) Search for map "a43b3b" - should say Sandbox v9 Beta Online Field Maps Web Map
![image](https://github.com/pjdohertygis/WideAreaSearchTemplates/assets/4256979/d4043de5-8cb4-4068-906c-14614c537dd7)
3) Open the map

**Complete the following steps to create a new task:** 
Step 1–Geolocate yourself, tap the blue Add button. 
![image](https://github.com/pjdohertygis/WideAreaSearchTemplates/assets/4256979/7b3a38ac-3637-41c5-adca-f5296d365923)

Step 2–Under Tasks, Tap the Unassigned task feature template.  

Step 3–Fill out the form and add the relevant information associated with the new task. 

Step 4–Tap Submit to save the new task 

Step 5-If you would like to assign the task to yourself, tap and open the unassigned task.  
![image](https://github.com/pjdohertygis/WideAreaSearchTemplates/assets/4256979/f1c4002d-9633-44dc-9a00-d6fa455080c1)

![image](https://github.com/pjdohertygis/WideAreaSearchTemplates/assets/4256979/657a1b4c-b484-453b-81ae-86da86a11c8f)

Step 6–Tap the Pick up feature action to assign it to yourself. 

**Tasking to a Person**
1) Find a buddy, task an assignment to them (if they aren't in the dropdown menu, have Jared or Paul add you)
2) Have them go to the assignment and complete it.

**Comment by susic1255** [comment]
Installed and was able to create a task and pick it up.  I started it and then Finished it with no issues.   If you want to drop some tasks and assign to me, I can see how the push notifications work.

**Comment by pjdohertygis** [comment]
@susic1255 added a point for you. I'm also testing out the "Integration" action which allows us to easily link out to any chat that allows for deep links. 
![image](https://github.com/pjdohertygis/WideAreaSearchTemplates/assets/4256979/15a6d9f1-e61f-49f4-8a32-02f1d4975a5a)

**Comment by stevefierro** [comment]
Installed and played with the interface.  Some items I noticed so far:

1. @pjdohertygis I had no issues with the date assignment popping up, selecting my due date, and continuing filling out the fields (I believe this is an apple thing that we have see even when using QC and S123)
2. Associating the task with a waypoint:  A) It asks for the waypoint object ID.  You can't get this when you are already in Fieldmaps and attempting to create a new task.  I can see the waypoint already there, but I can't select it to obtain the waypoint ID.  I have to select the associated waypoint prior to creating a new task, then obtain the ID (which I can copy).  Now I can go in and create a new task.  B) What does this actually do for us? The task information doesn't seem to have any reference to the waypoint.  I would have expected to be able to link to the information of the waypoint from within the task (when there is an association).
3. Attach does not appear to be working.  When I've started a task, the "MORE" spot has the ability to to "attach".  When selected, I get no pop up or anything allowing me to access a picture on my device or my camera.

I am using a Samsung Galaxy S22 with Android 14.

**Comment by stevefierro** [comment]
As a side note, you spelled my last name wrong :)

**Comment by pjdohertygis** [comment]
> As a side note, you spelled my last name wrong :)

Fixed!

**Comment by pjdohertygis** [comment]
@susic1255  asked "I did notice that there was no push notification of any type when I was assigned a task.  I assume that is because of the Beta app through Test Flight?"

@jdokemaps  Let's check with the EAC or Jeff Shaner / Doug M.

**Comment by pjdohertygis** [comment]
Reference this via https://github.com/pjdohertygis/SARCOP/issues/291 but closing to reduce confusion.
